features/all-positions/PositionActionsDialog.tsx

```diff
@@ -0,0 +1,588 @@
+                          />
+                        </Grid>
+                        <Grid item xs={6}>
+                          <TextField
+                            fullWidth
+                            type="number"
+                            inputProps={{ min: "0" }}
+                            label={`Max collateral/token`}
+                            value={maxCollPerToken}
+                            onChange={(
+                              e: React.ChangeEvent<HTMLInputElement>
+                            ) => setMaxCollPerToken(e.target.value)}
+                          />
+                        </Grid>
+                        <Grid item xs={6}>
+                          <TextField
+                            fullWidth
+                            type="number"
+                            inputProps={{ min: "0" }}
+                            label={`Max tokens to liquidate`}
+                            error={tokenBalanceTooLow}
+                            helperText={
+                              tokenBalanceTooLow
+                                ? `${tokenSymbol} balance too low`
+                                : null
+                            }
+                            value={maxTokensToLiquidate}
+                            onChange={(
+                              e: React.ChangeEvent<HTMLInputElement>
+                            ) => setMaxTokensToLiquidate(e.target.value)}
+                          />
+                        </Grid>
+                        <Grid item xs={6}>
+                          <FormControl fullWidth>
+                            <InputLabel>Deadline</InputLabel>
+                            <Select
+                              value={deadline}
+                              onChange={(
+                                e: React.ChangeEvent<{ value: unknown }>
+                              ) => setDeadline(e.target.value as string)}
+                            >
+                              <MenuItem value={60}>1 minute</MenuItem>
+                              <MenuItem value={5 * 60}>5 minutes</MenuItem>
+                              <MenuItem value={30 * 60}>30 minutes</MenuItem>
+                              <MenuItem value={60 * 60}>1 hour</MenuItem>
+                              <MenuItem value={5 * 60 * 60}>5 hours</MenuItem>
+                              <MenuItem value={1911772800}>Forever</MenuItem>
+                            </Select>
+                          </FormControl>
+                        </Grid>
+                      </Grid>
+                    </Box>
+
+                    <Box textAlign="center" pt={2} pb={2}>
+                      <Button
+                        fullWidth
+                        variant="contained"
+                        onClick={executeLiquidation}
+                        disabled={
+                          !(
+                            minCollPerToken &&
```

Comment:
Should these be `minCollPerTokenNum > 0 && maxCollPerTokenNum > 0 && etc...`?

Reply:
na, I dont think so. we dont have the `Num` versions of those for the input types, only for variables that are read in from other state containers. (i.e there is _no_ `minCollPerTokenNum` or `maxCollPerTokenNum`)